modules/serverless-rn-1-2-0.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-2-0_{context}"]
+//update the <version> to match the filename
+
+= Release Notes for Red Hat {ServerlessProductName} Technology Preview 1.2.0
+// add a version, e.g. Technology Preview 1.0.0
+
+[id="new-features-1-2-0_{context}"]
+== New features
+* {ServerlessProductName} has been updated to use Knative Serving 0.9.0.
+* {ServerlessProductName} has been updated to use Knative Client (`kn` CLI) 0.9.0.
+* {ServerlessProductName} on {product-title} 4.2 now uses the Operator Lifecycle Manager dependency resolution mechanism to install the ServiceMesh operator automatically. The required ServiceMeshControlPlane and ServiceMeshMemberRoll are also installed and managed for the user.
+* Access to the KnativeServing resource is now restricted to `cluster-admin` roles to prevent any user from blocking the resource. Only `cluster-admin` roles can create KnativeServing CRs.
+* The {ServerlessProductName} Operator can now be found in the OperatorHub by searching for "knative".
+* The {product-name} web console now shows status conditions for the KnativeServing resource.
+* In version 1.2.0, the {ServerlessProductName} Operator inspects network policies for namespaces.
++
+If no network policy exists, the Operator will automatically create a wide open policy, to ensure that traffic can flow in and out of the namespace and OpenShift routes can be used.
```

Comment:
`and OpenShift routes can be used.` -> `and OpenShift routes can still be used after the namespace is added to a ServiceMeshMemberRoll.`